python/cudf/cudf/tests/test_binops.py

```diff
@@ -1741,8 +1763,8 @@ def test_binops_with_lhs_numpy_scalar(frame, dtype):
     else:
         val = cudf.dtype(dtype).type(4)
 
-    expected = val == data.to_pandas()
-    got = val == data
+    expected = data.to_pandas() == val
+    got = data == val
```

Comment:
Not sure why these changed as part of this PR, but sure.

Reply:
Yes, this explanation is correct. The problem cases are when scalar is a numpy scalar and data is a cudf object; comparing `scalar == array` should work just fine. Can you add a comment in the code so that nobody accidentally flips this back? Then we should be good to go from my perspective.

Reply:
```suggestion
    # Compare equality with series on left side to dispatch to the pandas/cudf
    # __eq__ operator and avoid a DeprecationWarning from numpy.
    expected = data.to_pandas() == val
    got = data == val
```